packages/core/src/css/cssConfig.ts

```diff
@@ -79,35 +79,53 @@ export function isCssGlobalFile(
 
 type ExternalCallback = (arg0?: undefined, arg1?: string) => void;
 
-export function cssExternalHandler(
+export async function cssExternalHandler(
   request: string,
   callback: ExternalCallback,
   jsExtension: string,
   auto: CssLoaderOptionsAuto,
-  isStyleRedirect: boolean,
-): void | false {
-  const isCssModulesRequest = isCssModulesFile(request, auto);
-
+  styleRedirectPath: boolean,
+  styleRedirectExtension: boolean,
+  redirectPath: (request: string) => Promise<string | undefined>,
+): Promise<false | void> {
   // cssExtract would execute the file handled by css-loader, so we cannot external the "helper import" from css-loader
   // do not external @rsbuild/core/compiled/css-loader/noSourceMaps.js, sourceMaps.js, api.mjs etc.
   if (/compiled\/css-loader\//.test(request)) {
     return callback();
   }
 
+  let resolvedRequest = request;
+
+  if (styleRedirectPath) {
+    const redirectedPath = await redirectPath(request);
+    if (redirectedPath === undefined) {
```

Comment:
request of 3rd packages will return here earlier than before, which means this function not only deal with css externalize?